### PR TITLE
11.0 mig l10n ch zip

### PR DESCRIPTION
**Pull request by TDu**
This depends on:

- [ ] https://github.com/OCA/partner-contact/pull/488

### CODE REVIEW

File: l10n_ch_zip/README.rst [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+To use this module, you need to:
+
+* Fill the new field in the partner form with a zip or a city from Switzerland
+* Than you get a list with possible entries.
+* The one you select is auto-filled in the zip-, citiy-, state- and country-field.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/125/10.0
```
Comment (yvaucher) [inline l10n_ch_zip/README.rst]: s/10.0/11.0

File: l10n_ch_zip/__init__.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
```
Comment (yvaucher) [inline l10n_ch_zip/__init__.py]: Please remove this line in all files

In Python 3 the file is by default in utf-8

**Review by yvaucher — CHANGES_REQUESTED** [review]
2 minor changes to do

**Comment by TDu** [comment]
Fixes applied

**Pull request closed without merging**